src/applications/vaos/appointment-list/components/ConfirmedAppointmentDetailsPage/StatusAlert.jsx

```diff
@@ -134,7 +157,7 @@ StatusAlert.defaultProps = {
   appointment: {
     status: 'booked',
     cancelationReason: '',
-    avsPath: '',
+    avsPath: null,
```

Comment:
Why are default properties and appointment shape needed? The appointment shape is defined in the transformer as well as defaults. I think this pattern will be a problem to maintain when the shape changes.

Reply:
@vbahinwillit - Using shape for our props makes our data validations more thorough and accurate. I think we gain a lot from validating the types inside of a prop object. This makes it easier to determine what fields are required when debugging a component.